w3c/aria-at-app
# Add a new simple reports

#### Description
- This a new simplified version of the new reports page. 
- Based on these ["wireframes"](https://docs.google.com/spreadsheets/u/1/d/1jsvhue5xRxO03vvVxoqfljd_eihZcxKcz6HkhbSioD8/edit#gid=0) created after out discussion about redesigning this page with PAC.
- I reused and stripped down most of the code from the original, complex new reports page. The most important thing I preserved the marshalled data structure that we use to iterate over to show results. In the medium term I think it is a good solution, but it the long term it is not going to performant to be loading so much data that we don't need on any one page. 
- Remove the old main test report page `/results` but left the by run reports pages in place
- Fixed a bug so that runs with `updated_at = NULL` don't always show up as the most recently published version.
- This has been tested with NVDA on Windows. 

#### Follow up work
Tickets have already been for the following: 
- Adding breadcrumbs
- Handling invalid testReportIds in the URL and redirecting to a 404 page
- Adding the current test version info to the pages
- Adding links to the design page and examples to the example reports pages

#### QA Instructions
- Ensure you have some published test results
- Click on the "Test Reports" in the header or you can go there directly without signing in via by going to `/reports`.
